src/javascript/app_2/Utils/URL/url-helper.js

```diff
@@ -10,8 +10,7 @@ export default class URLHelper {
      */
     static getQueryParams(url) {
         const query_string =  url ? new URL(url).search : window.location.search;
-        const query_encoded = encodeURIComponent(query_string);
```

Comment:
What was the reason for the encode again? @cakasuma

Reply:
@OskarAhl @cakasuma thanks, i've added a fix